draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2883,7 +2889,125 @@ https://video.example.com/k/movie1/4/013.ts
          ]]></artwork>
           </figure>
         </section>
-      </section>
+	</section>
+	<section anchor="execution-policy" title="Execution Extension">
+	<t>
+		Unless specified otherwise, dCDN is at liberty to decide how to choose trigger commands for execution from all pending commands, whether to process trigger commands sequentially or in parallel, immediately upon acceptance, or with a delay in batches. A uCDN may wish to control trigger processing in more detail, including order of execution, dependencies, and concurrency.
+	</t>
+	<t>
+	Example use cases:
+		<list style="symbols">
+		<t>
+		Priority:  uCDN may have multiple trigger commands in “pending” and/or “active” mode. The trigger commands may potentially take a long time to execute, for example due to policy constraints, large number of content objects affected by it or other dCDN business logic.
```

Comment:
The second sentence is a little difficult to read.  Maybe:  "For example, trigger commands with policy constraints, large numbers of affected content objects, or other dCDN business logic may take a long time to execute."

Reply:
Accepted